### PR TITLE
Docstring for Series.min() and Series.mean()

### PR DESCRIPTION
**Pull request by ugohuche**

### CODE REVIEW

File: narwhals/series.py [hunk 1]
```diff
@@ -99,6 +99,30 @@ def cast(
...
+            >>> s_pd = pd.Series(s)
+            >>> s_pl = pl.Series(s)
+
+            We define a library agnostic function:
+
+            >>> def func(s_any):
+            ...     s = nw.from_native(s_any, series_only=True)
+            ...     return s.mean()
+
+            We can then pass pandas, Polars or any other library Series to `func`:
```
Comment (MarcoGorelli) [inline narwhals/series.py:119]: 'any other library' might be a bit too much 😄 shall we just say "any other Narwhals-compliant library"? or just leave it as "pandas or Polars" and we'll sort out the "any other library" part later in a large pull request

Comment (ugohuche) [reply narwhals/series.py:119]: Right... 😅

Comment (ugohuche) [reply narwhals/series.py:119]: Okay

**Review by MarcoGorelli — APPROVED** [review]
thanks @ugohuche !

**Comment by ugohuche** [comment]
Sure thing @MarcoGorelli. Thanks for the GIF 😊

**Pull request merged**